test/result/function/func_aggr_avg.result

```diff
@@ -31,20 +31,15 @@ select avg(h) from t1;
 avg(h)
 14.31400000000000000000000
 select avg(i) from t1;
-avg(i)
-20120312.0000
+Function 'avg' with parameters [DATE] will be implemented in future version.
```

Comment:
You shouldn't modify the expected result, if new errors are introduced, you need to add new tags for it.